### ISSUE TITLE
Error when trying to rename a folder in a dropbox location.

### ISSUE THREAD

**Issue by GrfxGuru** [careteditor/caret#562]
I was attempting to rename a folder in the left hand tree view. Username removed from location.

The following error has occurred:
```
{
    "message": "EPERM: operation not permitted, rename 'C:\\Users\\...\\Dropbox (Personal)\\Markdown Files\\Swift 3 Book' -> 'C:\\Users\\...\\Dropbox (Personal)\\Markdown Files\\Swift 4 Book'",
    "version": "3.4.6",
    "os": "Windows 10",
    "path": "fs.js",
    "line": 776,
    "column": 18,
    "stack": [
        "at Object.fs.renameSync ((fs.js:776:18)",
        "at e.value ((C:\\Users\\...\\AppData\\Local\\Caret\\app-3.4.6\\resources\\app.asar\\extensions\\FileSystem.js:5:1516)",
        "at Object.callback ((C:\\Users\\...\\AppData\\Local\\Caret\\app-3.4.6\\resources\\app.asar\\extensions\\FilesSidebar.js:5:8641)",
        "at t.value ((C:\\Users\\...\\AppData\\Local\\Caret\\app-3.4.6\\resources\\app.asar\\web\\Events.js:5:1335)",
        "at C:\\Users\\...\\AppData\\Local\\Caret\\app-3.4.6\\resources\\app.asar\\web\\rename.js:5:2715"
    ]
}
```

**Comment by astoilkov** [comment]
Hi @GrfxGuru,

We think we fixed the problem. The fix will be included in the next Caret release. I will let you know when that happens so you could try it.

Thanks for reporting this problem.

**Comment by astoilkov** [comment]
This is now fixed in the new Caret 4.0 Beta update available [here](https://caret.io/4beta/).

Closing this issue. Thanks for reporting.
